graphs/adapter-api/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/api/GraphBehaviours.kt

```diff
@@ -29,3 +32,15 @@ open class AddEdgeBehaviour : ReifiedBehavior {
 open class GetShortestPathBehaviour : ReifiedBehavior {
     override val behavior = GraphBehaviours.GetShortestPath
 }
+
+open class HasCycleBehavior : ReifiedBehavior {
+    override val behavior = GraphBehaviours.HasCycles
+}
```

Comment:
🚫 The choice of using singular (`HasCycle`) or plural (`HasCycles`) should be consistent throughout. You previously expressed preference for the former, but now most (but not all) places seem to use the latter...